pkg/api/api.go

```diff
@@ -130,18 +152,40 @@ func (dvol *DvolAPI) CreateBranch(volumeName, branchName string) error {
 	return dvol.dl.CreateVariant(volumeName, branchName)
 }
 
-func (dvol *DvolAPI) SwitchBranch(volumeName, branchName string) error {
-	return nil
+var NotFound = errors.New("Item not found")
+
+func indexOfVariant(variantName string, variants []string) (int, error) {
```

Comment:
Is there a reason that this function returns the index of a variant when it is just used to check whether a given variant exists?